app/_src/gateway/migrate-to-new-dns-client.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Migrate to the new DNS client
```

Comment:
I think maybe this would fit better in the Networking section of the docs (same section as https://deploy-preview-7805--kongdocs.netlify.app/gateway/latest/production/networking/dns-considerations/#main).

I can't decide if I think this would be better if it was added to that existing DNS doc. I searched for CNAME and SRV in that other DNS doc and didn't come up with anything. Is there something we could add from this doc to that one?

Reply:
> maybe we should have both? I could add a mention of the two clients in the DNS considerations section.

I think this sounds good to me!